### ISSUE TITLE
Issue with outdated mongodb version

### ISSUE THREAD

**Issue by nsine** [agenda/agenda#918]
Hey 👋
After mongodb is upgraded to v3.5.x - many packages started using it. However `agenda` still uses old v3.4.1. This results in some incompatibility issues between these versions. And e.g. in our project we have this error right after start:
```
  if (s.topology.connections().length === 0) {
                 ^
TypeError: s.topology.connections is not a function
    at Agenda.module.exports [as _findAndLockNextJob] (/xx/node_modules/agenda/lib/agenda/find-and-lock-next-job.js:25:18)
    at jobQueueFilling (/xx/node_modules/agenda/lib/utils/process-jobs.js:170:10)
    at Agenda.module.exports (/xx/node_modules/agenda/lib/utils/process-jobs.js:32:9)
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
```
There is already a PR from renovate bot for upgrading version https://github.com/agenda/agenda/pull/910. Can we please do a patch release with updated `mongodb` if there are no any issues with it?

**Comment by nsine** [comment]
https://github.com/agenda/agenda/issues/919

**Comment by MichielDeMey** [comment]
Should be fixed in `v3.0.0`. 👍
